### DISCUSSION TITLE
Running conductor-server as a spring boot app

### DISCUSSION THREAD

**Discussion by br3nchaa** [Netflix/conductor#3373] [Q&A]
Hello,

I am trying to use the conductor-server binary available in maven repository as a dependency in my gradle project. However, when I start up the app, the swagger url has none of the api definitions and just says "No operations defined in spec!". 

Is there another dependency I should be adding or configuration of some sort? 
<img width="1440" alt="Screen Shot 2022-12-06 at 6 14 58 PM" src="https://user-images.githubusercontent.com/39531320/206054284-780829f7-75fa-440e-8045-4b590991d47e.png">
<img width="1434" alt="Screen Shot 2022-12-06 at 6 15 28 PM" src="https://user-images.githubusercontent.com/39531320/206054332-1a4b14e8-2eb4-4a7c-a193-3a09abc33e64.png">

**Comment by manan164** [comment]
Hi @br3nchaa , I will check and update.

**Comment by v1r3n** [comment]
Hi @br3nchaa can you share your properties file you used to start conductor server.
Can you share the output of `http://localhost:8080/api-docs` to see what is the response?

**Reply by br3nchaa** [reply]
<img width="601" alt="Screen Shot 2022-12-06 at 6 14 39 PM" src="https://user-images.githubusercontent.com/39531320/206054368-bab2dbc1-1a2a-4a0c-8ed5-26249b754d53.png">

**Comment by ceroy-ak** [comment]
> Hello,
> 
> I am trying to use the conductor-server binary available in maven repository as a dependency in my gradle project. However, when I start up the app, the swagger url has none of the api definitions and just says "No operations defined in spec!". 
> 
> Is there another dependency I should be adding or configuration of some sort? 
> <img width="1440" alt="Screen Shot 2022-12-06 at 6 14 58 PM" src="https://user-images.githubusercontent.com/39531320/206054284-780829f7-75fa-440e-8045-4b590991d47e.png">
> <img width="1434" alt="Screen Shot 2022-12-06 at 6 15 28 PM" src="https://user-images.githubusercontent.com/39531320/206054332-1a4b14e8-2eb4-4a7c-a193-3a09abc33e64.png">
> 
> 
> 
> 

@br3nchaa were you able to solve the issue? If so then please share the steps.

Also I'm trying to work with Maven, so in pom.xml what all dependencies should I add to run the conductor from a sprint boot application
